packages/nextjs/app/builders/page.tsx

```diff
@@ -0,0 +1,77 @@
+"use client";
+
+import Link from "next/link";
+import type { NextPage } from "next";
+import { MagnifyingGlassPlusIcon } from "@heroicons/react/24/outline";
+import { Address } from "~~/components/scaffold-eth";
+import { useScaffoldEventHistory } from "~~/hooks/scaffold-eth";
+
+const Builders: NextPage = () => {
+  const { data: checkedInEvents, isLoading: isCheckedInEventsLoading } = useScaffoldEventHistory({
+    contractName: "BatchRegistry",
+    eventName: "CheckedIn",
+    fromBlock: 120936875n,
+  });
+
+  return (
+    <>
+      <div className="flex items-center flex-col flex-grow pt-10">
+        <div className="px-5">
+          <h1 className="text-center">
+            <span className="block text-4xl font-bold">Get to know Batch 6 builders!!</span>
+          </h1>
+        </div>
+        <div className="flex items-center flex-col flex-grow">
+          {isCheckedInEventsLoading ? (
```

Comment:
Ohh its seems there is some bug with SE-2 `useScaffoldEvenHistory` hook the `isCheckedInEventsLoading` constantly flicker and causes this : 


https://github.com/BuidlGuidl/batch6.buidlguidl.com/assets/80153681/c8147483-4f48-47ab-ad9a-97416b31a5a6

For now lets remove the use of `ischeckedInEventLoading` and rely on just `!checkedInEvents || checkedInEvents.length === 0`